pkg/steps/artifacts.go

```diff
@@ -340,6 +342,30 @@ func removeFile(podClient PodClient, ns, name, containerName string, paths []str
 	return nil
 }
 
+func addPodUtils(pod *coreapi.Pod, artifactDir string, decorationConfig *v1.DecorationConfig, rawJobSpec string) error {
```

Comment:
Could you add/extend a fixture-based test for this somewhere? Seeing this as yaml makes reasoning much easier than just the code

Reply:
done